pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```
@@ -17,19 +18,31 @@
     extraMeta = {
       branch = "master";
       maintainers = with lib.maintainers; [ davidak Madouura ];
-      broken = true;
     };
   } // argsOverride;
 
-  kernelPatches = [ {
+  kernelPatches = [
+    {
       name = "bcachefs-${commit}";
 
       patch = fetchpatch {
         name = "bcachefs-${commit}.diff";
         url = "https://evilpiepirate.org/git/bcachefs.git/rawdiff/?id=${commit}&id2=v${lib.versions.majorMinor kernel.version}";
         sha256 = diffHash;
+
+        postFetch = ''
+          ${pkgs.buildPackages.patchutils}/bin/filterdiff -x 'a/block/bio.c' "$out" > "$tmpfile"
```

```suggestion
          filterdiff -x 'a/block/bio.c' "$out" > "$tmpfile"
```
that should be in stdenv

---

It is, but there is no way to get it to work in this derivation it seems.